planning/behavior_velocity_planner/CMakeLists.txt

```diff
@@ -56,6 +56,12 @@ ament_auto_add_library(scene_module_lib SHARED
   src/utilization/interpolate.cpp
 )
 
+target_include_directories(scene_module_lib
+  SYSTEM PUBLIC
```

Comment:
Why do you add `PUBLIC` for only a few packages? What is the difference between the packages that you didn't add it?

Reply:
`PUBLIC/PRIVATE/INTERFACE` keyword is necessary for `target_include_directories`.
https://cmake.org/cmake/help/latest/command/target_include_directories.html

Reply:
Ah, I see, thank you!